crates/typst-library/src/foundations/calc.rs

```diff
@@ -933,6 +932,8 @@ pub fn rem(
 /// The result of this computation is that of a division rounded to the integer
 /// `{n}` such that the dividend is greater than or equal to `{n}` times the divisor.
 ///
+/// This can error if the dividend is too small in magnitude compared to the divisor.
```

Comment:
Seems like this needs an update now.
```suggestion
/// This can error if the dividend is too small in magnitude compared to the
/// divisor, or if the result of integer division would be larger than the
/// maximum 64-bit signed integer.
```

Reply:
Good catch, thanks!